.github/workflows/main.yml

```diff
@@ -103,30 +75,10 @@ jobs:
           CCX: ${{ matrix.config.cxx }}
           FILE_MEMORY: ${{ matrix.config.file_memory }}
           BUILD_TYPE: ${{ matrix.config.build_type }}
-          COVERAGE: ${{ matrix.config.coverage }}
+          COVERAGE: OFF
```

Comment:
These variables are not the right input for `ccache-action` anyway, as it only uses `key` variable which defines the cache ID. Something is wrong with this step, it's a mix of 2 steps, in fact.